charon/utils/logs.py

```diff
@@ -72,3 +72,12 @@ def set_logging(name="charon", level=logging.DEBUG, handler=None):
     logger = logging.getLogger('charon')
     for hdlr in list(logger.handlers):  # make a copy so it doesn't change
         hdlr.setFormatter(formatter)
+
+
+def add_file_handler(logger: logging):
+    handler = logging.FileHandler('errors.log')
```

Comment:
Can we make this "errors.log" configurable for different location? I think we can configure this in CharonConfig, and give it a default location like this "./errors.log".

Reply:
@ligangty If we introduce the CharonConfig in logs.py, it will cause the error of partially initialized, it's likely a circular import issue, otherwise, we could use another config(or log properties) rather than CharonConfig instead.

Reply:
Or maybe we can let the config.py not use this getLogger method, as the error in CharonConfig is not our target to record in the errors.log.